junit-jupiter-api/src/main/java/org/junit/jupiter/api/DisplayNameGenerator.java

```diff
@@ -92,6 +92,9 @@ static String parameterTypesAsString(Method method) {
 
 		static final DisplayNameGenerator INSTANCE = new Standard();
 
+		public Standard() {
+		}
```

Comment:
I assume that this constructor is needed by `-Werror`, right?

Reply:
Yes, due to newly introduced warnings like these: https://ge.junit.org/s/aohc47ea6wfk6/console-log?task=:junit-jupiter:compileModule